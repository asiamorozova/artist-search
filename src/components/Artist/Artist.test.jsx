import React from 'react';
import { shallow } from 'enzyme';
import Artist from './Artist';

describe('Artist', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<Artist
      id='string'
      name={'some artist'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
