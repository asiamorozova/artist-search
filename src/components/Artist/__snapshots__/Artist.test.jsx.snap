// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Artist matches a snapshot 1`] = `
<h3>
  some artist
</h3>
`;
